spot_rl_experiments/spot_rl/utils/img_publishers.py

```diff
@@ -547,12 +550,18 @@ def _publish(self):
 
         # publish data
         self.publish_new_detection(";".join(object_info))
+        self.publish_viz_img(viz_img, header)
 
         stopwatch.print_stats()
 
     def publish_new_detection(self, new_object):
         self.pubs[self.detection_topic].publish(new_object)
 
+    def publish_viz_img(self, viz_img, header):
```

Comment:
so this will overwrite the existing image publisher?

Reply:
This will not overwrite existing image publisher. It will publish on the same topic.